wallet/wallet.go

```diff
@@ -1622,6 +1622,38 @@ func (w *Wallet) AccountAddresses(account uint32) (addrs []btcutil.Address, err
 	return
 }
 
+// AccountManagedAddresses returns the managed addresses for every created
+// address for an account.
+func (w *Wallet) AccountManagedAddresses(scope waddrmgr.KeyScope,
```

Comment:
Solid change! It just so happens that lnd is using the exact same `walletdb` abstraction for its database here. In the future, this won't be the case, so this is also a prep for when we move away from bolt in this package.